integration/scripts/deploy.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2022 Intel Corporation
+#
+# This script is used to deploy the OPI PoC into one of two
+# environments:
+#
+# * Developer emulated xPU setup
+# * Split setup between an actual xPU and a host/VM/container
+#
+
+set -eo pipefail
+
+# Enable debug if running in CI
+if [ -n "${CI}" ] && [ "${CI}" == "true" ]
+then
+    set -x
+fi
+
+# DOCKER_COMPOSE setup
+DC=docker-compose
+
+if [ "$(command -v ${DC})" == "" ]
+then
+    DC="docker compose"
+fi
+
+
+usage() {
+    echo ""
+    echo "Usage: $0 -m [dev | xpu] -i [IP address of the host] -x [IP address of the xPU]"
+    echo ""
+    exit 1
+}
+
+deploy_dev() {
+    echo "Deploying development emulated environment"
+
+    # Run the integration script to start things up
+    ./scripts/integration.sh start
+}
+
+deploy_xpu() {
+    echo "Deploying xPU environment"
+    echo "Host IP address: ${HOST_IP_ADDRESS}"
```

Comment:
by Host you mean you local VM/laptop or actual host ?

Reply:
I mean the place to run all the things in the upper right of the integration diagram (e.g. PXE, tftp, SPDK target, etc.

Reply:
sure, let's just not use `host` term for this...

`host` should be reserved for the server where DPU is plugged into... running user apps/contaienrs/vms with real workload

Reply:
Suggestions on a name? What about `infra` or `infra_host`?

Reply:
yep, `infra` is good, and let's put a default to be `127.0.0.1` if you running this on your laptop